recipes/trackhub/0.1.2019.12.24/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "trackhub" %}
+{% set version = "0.1.2019.12.24" %}
+{% set commit = "157ac4d3c392bd8c47b675a68f3c1a25b08a4afa" %}
+{% set sha256 = "4693509121d7e3eb2d2750a59af6b3fdfd391cc90519a9931add77aa588f80e3" %}
```

Comment:
The sha256sum is `a13b59e942d94bd6bdcb95c6cd480717224641f8e66400581d4ffa3bc7732874`